Formula/sshguard.rb

```diff
@@ -3,7 +3,7 @@ class Sshguard < Formula
   homepage "https://www.sshguard.net/"
   url "https://downloads.sourceforge.net/project/sshguard/sshguard/2.2.0/sshguard-2.2.0.tar.gz"
   sha256 "2aff07fee6ec33e4ffd5411916b75189977af1d77b86dac5f3834dd3aa3656c2"
-  version_scheme 1
+  version_scheme 2
```

Comment:
This isn't doing what you think it is 😉. If you're trying to bump the revision you want a `revision 1` line. Somewhat hesitant to bump the revision for such a minor change given the lack of people complaining about this issue though. Open to other people's opinions 🤷‍♂️.

Reply:
Should I change the pull request, or will the maintainer correct that?

The sshguard recipe often goes for long periods of time in a non-working state, for example when macOS changed the logging system. I think there is a low number of users, or at least users that check up on whether it's actually working or not.

Reply:
You'll need to do it. Let's skip the `revision` addition though unless more people complain about it; anyone who wants `brew` to remove the slashes rather than doing it themselves can just `brew reinstall sshguard` after this PR is merged.

Reply:
Alright. Can you give me some direction on what I need to do? Like, "remove the version_scheme bump and resubmit the pull request".

Reply:
Return this line to reading `version_scheme 1` & then push that change to this branch. Advice on doing that, if required, depends on whether you're using the web UI or the `git` CLI.